outputs/elasticsearch.go

```diff
@@ -73,6 +130,45 @@ func (c *Client) ElasticsearchPost(falcopayload types.FalcoPayload) {
 	c.PromStats.Outputs.With(map[string]string{"destination": "elasticsearch", "status": OK}).Inc()
 }
 
+func (c *Client) ElasticsearchCreateIndexTemplate(config types.ElasticsearchOutputConfig) error {
+	d := c
+	var err error
+	u, err := url.Parse(fmt.Sprintf("%s/_index_template/falco", config.HostPort))
+	if err != nil {
+		log.Printf("[ERROR] : %v - %v\n", c.OutputType, err.Error())
+		return err
+	}
+	d.EndpointURL = u
+	if err := d.Get(); err != nil {
```

Comment:
this block has a very deep nesting, you could think about a dedicated function which uses early returns instead

Reply:
Got it. Just changed the func following your comment